projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quick-order.ts

```diff
@@ -0,0 +1,105 @@
+import { SampleProduct } from '../../sample-data/checkout-flow';
+import * as sampleData from '../../sample-data/b2b-checkout';
+import { verifyTabbingOrder as tabbingOrder } from '../accessibility/tabbing-order';
+import { tabbingOrderConfig as config } from '../../helpers/accessibility/b2b/tabbing-order.config';
+import { waitForPage } from '../checkout-flow';
+
+export const ADD_TO_CART_ENDPOINT_ALIAS = 'addEntry';
+
+export function interceptAddToCartEndpoint() {
+  cy.intercept(
+    'POST',
+    `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+      'BASE_SITE'
+    )}/orgUsers/*/carts/*/entries*`
+  ).as(ADD_TO_CART_ENDPOINT_ALIAS);
+
+  return ADD_TO_CART_ENDPOINT_ALIAS;
+}
+
+export function visitCartPage() {
+  const cartPage = waitForPage('/cart', 'cartPage');
+
+  cy.visit(`/cart`);
+  cy.wait(`@${cartPage}`).its('status').should('eq', 200);
+}
+
+export function visitQuickOrderPage() {
+  const quickOrderPage = waitForPage(
+    '/my-account/quick-order',
+    'quickOrderPage'
+  );
+
+  cy.visit('/my-account/quick-order');
+  cy.wait(`@${quickOrderPage}`).its('status').should('eq', 200);
+}
+
+export function addProductToTheList(productCode: string) {
+  cy.get('.quick-order-form-input input').type(`${productCode}{enter}`);
+}
+
+export function fillFormWithProductCode(product: SampleProduct) {
+  cy.get('.quick-order-form-input input').type(`${product.code}`);
+}
+
+export function clearForm() {
+  cy.get(`.quick-order-form-input cx-icon`).click();
+}
+
+export function clearList() {
+  cy.get(`.quick-order-footer .clear-button`).click();
```

Comment:
it would be great to add after this line to assert the global error message being thrown that it's being cleared

Reply:
```suggestion
  cy.get(`.quick-order-footer .clear-button`).click();
  alerts.getSuccessAlert().should('contain', `Quick order list has been cleared`);
```

from import * as alerts from helpers/global-message